README.md

```diff
@@ -267,6 +267,7 @@ used in the code. Introduction and/or reference of those will be provided in the
     <td align="center"><a href="https://github.com/mover333"><img src="https://avatars.githubusercontent.com/u/55155635?s=460&v=4" width="100px;" alt=""/><br /><sub><b>Mitchell Overfield</b></sub></a><br /></td>
     <td align="center"><a href="https://github.com/alkofahi"><img src="https://avatars.githubusercontent.com/u/58956796?s=460&v=4" width="100px;" alt=""/><br /><sub><b>Jafar Al-Kofahi</b></sub></a><br /></td>
```

Comment:
You also need to add a new section to this root README. Look at the End to end demos ✨
NEW (▀̿Ĺ̯▀̿ ̿) section, basically copy that and call it PowerApps and feel free to add a new icon to it :)

Reply:
I'll add it tonight.